facturacion_mexico/facturacion_fiscal/tests/test_tipo_e_nota_credito.py

```
@@ -0,0 +1,197 @@
+
+from unittest.mock import MagicMock, patch
+
+import frappe
+from frappe import _
+from frappe.tests.utils import FrappeTestCase
+
+
+# ── helpers ──────────────────────────────────────────────────────────────────
+
+
+def _mock_ffm(fm_tipo_comprobante="E - Egreso", fm_uuid_relacionado="", fm_tipo_relacion_sat="01"):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_mock_ffm`

(ANN202)